### PR TITLE
Add lakectl abuse commit

### PR DESCRIPTION
**Pull request by itaiad200**
Part of #3571 

A command to repeatedly commit to the source branch. Together with `abuse random-write`, it grants the ability to benchmark lakeFS over KV/DB commit-during-write flow.

### CODE REVIEW

File: cmd/lakectl/cmd/abuse.go [hunk 1]
```diff
@@ -152,6 +153,53 @@ var abuseRandomWritesCmd = &cobra.Command{
...
+				resp, err := client.CommitWithResponse(ctx, u.Repository, u.Ref, &api.CommitParams{},
+					api.CommitJSONRequestBody(api.CommitCreation{Message: work}))
+				if err == nil && resp.StatusCode() != http.StatusOK {
+					err = helpers.ResponseAsError(resp)
+				}
+				output <- stress.Result{
+					Error: err,
+					Took:  time.Since(start),
+				}
+				time.Sleep(gapDuration)
```
Comment (nopcoder) [inline cmd/lakectl/cmd/abuse.go:197]: Support cancellation
```suggestion
				time.Sleep(gapDuration)
				select{
                                     case <-ctx.Done():
                                         return
                                     case <-time.After(gapDuration):
                                 }
```

File: cmd/lakectl/cmd/abuse.go [hunk 2]
```diff
@@ -152,6 +153,53 @@ var abuseRandomWritesCmd = &cobra.Command{
...
 }
 
+var abuseCommitCmd = &cobra.Command{
+	Use:    "commit <source ref uri>",
+	Short:  "Commits to the source ref repeatedly",
+	Hidden: false,
+	Args:   cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		u := MustParseRefURI("source ref", args[0])
+		amount := MustInt(cmd.Flags().GetInt("amount"))
```
Comment (nopcoder) [inline cmd/lakectl/cmd/abuse.go:163]: very amount > 0

**Review by nopcoder — APPROVED** [review]
LGTM! Added suggestions to enable cancellation and basic verification.

File: cmd/lakectl/cmd/abuse.go (hunk 2 above)
Comment (itaiad200) [reply cmd/lakectl/cmd/abuse.go:163]: It will run 0 commits.

**Pull request merged**